src/ansys/mapdl/core/mapdl_core.py

```diff
@@ -2261,7 +2263,7 @@ def run(
             # simply return the contents of the file
             return self.list(*command.split(",")[1:])
 
-        if "=" in command:
+        if "=" in command and not is_comment:
```

Comment:
Is it possible a comment contains a ``=``? It might be the reason of the ``check_vulnerability`` warning

Reply:
it is... a comment should not be evaluated in MAPDL, so it should contain anything.